package.json

```
@@ -50,10 +50,11 @@
     "@wordpress/notices": "2.13.3",
     "@wordpress/prettier-config": "^3.15.0",
     "@wordpress/primitives": "^3.0.0",
-    "@wordpress/scripts": "^13.0.2",
+    "@wordpress/scripts": "26.19.0",
     "@wordpress/url": "^3.2.1",
     "allure-playwright": "^2.0.0-beta.20",
     "archiver": "^5.0.0",
+    "axios": "1.12.2",
```

Do we want these as specific versions, or is it worth specifying them using `^`?
```suggestion
    "@wordpress/scripts": "^26.19.0",
    "@wordpress/url": "^3.2.1",
    "allure-playwright": "^2.0.0-beta.20",
    "archiver": "^5.0.0",
    "axios": "^1.12.2",
```